service_mesh/service_mesh_install/updating-ossm.adoc

```diff
@@ -4,7 +4,84 @@ include::modules/ossm-document-attributes.adoc[]
 :context: updating-ossm
 toc::[]
 
-If you are updating from {ProductName} 1.0 to 1.1, you must update the `ServiceMeshControlPlane` resource to update the control plane components to the new version. 
+
+== Manual updates required by version 1.1.4
+
+The fix for https://bugzilla.redhat.com/show_bug.cgi?id=1844254[CVE-2020-8663]`: envoy: Resource exhaustion when accepting too many connections` added a configurable limit on downstream connections. The configuration option for this limit must be configured to mitigate this vulnerability.
+
+This new configuration option is called `overload.global_downstream_max_connections`, and it is configurable as a proxy `runtime` setting.  Perform the following steps to configure limits at the ingress gateway.
+
+.Procedure
+
+. Create a file named `bootstrap-override.json` with the following text to force the proxy to override the bootstrap template and load runtime configuration from disk:
++
+  {
+    "runtime": {
+      "symlink_root": "/var/lib/istio/envoy/runtime"
+    }
+  }
++
+. Create a secret from the `bootstrap-override.json` file, replacing <SMCPnamespace> with the namespace where you created the service mesh control plane (SMCP):
++
+  oc create secret generic -n <SMCPnamespace> gateway-bootstrap --from-file=bootstrap-override.json
++
+. Update the SMCP configuration to activate the override.
+
++
+.Updated SMCP configuration example #1
+[source,yaml]
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshControlPlane
+spec:
+  istio:
+    gateways:
+      istio-ingressgateway:
+        env:
+          ISTIO_BOOTSTRAP_OVERRIDE: /var/lib/istio/envoy/custom-bootstrap/bootstrap-override.json
+        secretVolumes:
+        - mountPath: /var/lib/istio/envoy/custom-bootstrap
+          name: custom-bootstrap
+          secretName: gateway-bootstrap
+----
++
+
+. To set the new configuration option, create a secret that has the desired value for the `overload.global_downstream_max_connections` setting (the following example uses a value of `10000`):
```

Comment:
Rather than parentheses, I would make the last part a new sentence: "The following example..."